packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import styles from '@patternfly/patternfly/components/ClipboardCopy/clipboard-copy.css';
+import { css } from '@patternfly/react-styles';
+import { CopyIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { TextInput } from '../TextInput';
+import { Tooltip, TooltipPosition } from '../Tooltip';
+import GenerateId from '../../helpers/GenerateId/GenerateId';
+import CopyButton from './CopyButton';
+import ToggleButton from './ToggleButton';
+import ExpandedContent from './ExpandedContent';
+
+export const clipboardCopyFunc = (event, text) => {
```

Comment:
just wondering if this needs to be exported?  Is there a possibility it will be used outside of this class?

Reply:
Nope, this export is unnecessary. I removed it.
Thanks for noticing. :smile: